commands/inventory/list_services.go

```diff
@@ -108,11 +108,15 @@ func (cmd *listServicesCommand) Run() (commands.Result, error) {
 		})
 	}
 	for _, s := range result.Payload.Postgresql {
+		addressPort := net.JoinHostPort(s.Address, strconv.FormatInt(s.Port, 10))
```

Comment:
looks like it can be extracted to somewhere

Reply:
What do you think if we will add extra field for socket in `listResultService` struct and move this logic there?

Reply:
Yes, I agree - but going to do all cleanups in the PRs "socket for MongoDB" to avoid duplicates and merge conflicts.